test/dotnet/src/ScriptDatabase.cs

```diff
@@ -43,7 +44,8 @@ public static void ScriptDatabase(string strLine, string testName, TestUtils tes
 					NoCollation = true
 				}
 			};
-
+			testUtils.PrintToLogsOrConsole("Calling ScriptLogins...", logger, "debug");
+            ScriptLogins(server, scripter, testName, testUtils, logger);
```

Comment:
This statement should not be inside ScriptDatabase, since login is not database level. Consider calling it where ScriptDatabase is called.